Source/diablo.cpp

```diff
@@ -1547,7 +1544,7 @@ static void GetMousePos(LPARAM lParam)
 	MouseY = (short)((lParam >> 16) & 0xffff);
 }
 
-void DisableInputWndProc(UINT uMsg, WPARAM wParam, LPARAM lParam)
+void DisableInputWndProc(UINT uMsg, WPARAM /*wParam*/, LPARAM lParam)

```

Comment:
This is a really annoying rule :D I suggest removing it.

Reply:
skipped for now